MiniAn/minian_main.py

```diff
@@ -784,36 +784,28 @@ def save_minian_to_doric(
     res = Y - AC # residual images
 
     print(mn_defs.Messages.GEN_ROI_NAMES, flush = True)
-    names = []
-    usernames = []
-    for i in A.coords["unit_id"].values:
-        names.append("ROI"+str(i).zfill(4))
-        usernames.append("ROI {0}".format(i))
+    roi_ids           = A.coords["unit_id"].values
+    roi_dataset_names = [defs.DoricFile.Dataset.ROI.format(str(id_).zfill(4)) for id_ in roi_ids]
+    roi_usernames     = [defs.DoricFile.Dataset.ROI.format(id_) for id_ in roi_ids]
+    roi_names         = [defs.DoricFile.Dataset.ROI.format(id_) for id_ in roi_ids]
```

Comment:
I did something like that and them I feed to value to the functions that save the ROI and spikes. What do you think?